etcdserver/util.go

```diff
@@ -95,3 +96,19 @@ func (nc *notifier) notify(err error) {
 	nc.err = err
 	close(nc.c)
 }
+
+func loadBackend(bepath string, be *backend.Backend, QuotaBackendBytes int64) {
```

Comment:
can we just return `backend.Backend`?
And `s/QuotaBackendBytes/quotaBackendBytes/`?